### ISSUE TITLE
Minimal spatial coverage required for useful network measures

### ISSUE THREAD

**Issue by roschkoenig** [roschkoenig/Network_Analysis_for_Epilepsy_Surgery#2]
- [ ] **Define problem**
- [ ] **Literature summary** 
- [ ] **Proposed solution and dataset** 
- [ ] **Analysis scripts written** 
- [ ] **Summary of the results**

**Comment by gmschroe** [comment]
(Thought I'd use comments first to discuss the issue before proposing a solution...)

One approach for analyzing this might be to start with a patient with large spatial coverage and then gradually delete nodes and recompute the network measures for each subsequently smaller network.

The inferred epileptogenic zone (based on surgical outcome) could either be retained in the network throughout all iterations or arbitrarily deleted (so we know how networks missing the EZ will behave).

Patients needed: 
- good surgical outcome, large spatial coverage (grids/strips)
- good surgical outcome, many depth electrodes

We would need to define our connectivity measure and network measures of interest first, though.

**Comment by yujiangwang** [comment]
I agree, I think we might have to look at some other technical issues first.
I suspect that we may be better off looking at this issue at a later stage with scalp EEG, EEG/fMRI, or MEG data; and by epilepsy subtype. Not sure if there is a simple definition of minimal spatial coverage for all types of patients.
